lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -164,3 +164,323 @@ testRule(
 		],
 	}),
 );
+
+testRule({
+	ruleName,
+	config: ['lower'],
+	syntax: 'css-in-js',
+	fix: true,
+	accept: [
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	stroke: "url(#gradiantA)",
+});
+`,
+			description: 'href with location',
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "pink",
+});
+`,
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "#000",
+});
+`,
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	something: "#000, #fff, #ababab",
+});
+`,
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "#0000ffcc",
+});
+`,
+			description: 'eight digits',
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	color: "#00fc",
+});
+`,
+			description: 'four digits',
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled.a({
+	padding: "000",
+});
+`,
+		},
+		{
+			code: `
+import styled from 'styled-components';
+export const s = styled("a::before")({
+	content: '"#ABABA"',
```

Comment:
```suggestion
	content: '"#ABABAB"',
```

Let's use a valid hex.